<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>635</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FileManager{
	background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="BackgroundFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#BackgroundFrame{
	background-color: rgb(255, 204, 175);
	border-top-right-radius: 25px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>70</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="StackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="ExpandedPage">
         <property name="styleSheet">
          <string notr="true">#ExpandedPage{
	background-color: transparent;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>30</number>
            </property>
            <property name="leftMargin">
             <number>26</number>
            </property>
            <property name="topMargin">
             <number>30</number>
            </property>
            <item>
             <widget class="QPushButton" name="AnalyzeBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>55</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>55</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	image: url(:/images/analyze-big.svg);
	background-color: rgb(255, 155, 82);
	border-radius: 10px;
	padding-left: 20px;
}

QPushButton:hover{
	image: url(:/images/analyze-big-hover.svg);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="styleSheet">
               <string notr="true">#frame_2{	
	background-color: rgb(255, 245, 239);
	border-top-left-radius: 25px;
	border-top-right-radius: 25px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="leftMargin">
                  <number>15</number>
                 </property>
                 <property name="rightMargin">
                  <number>15</number>
                 </property>
                 <item>
                  <widget class="QFrame" name="frame_3">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">#frame_3{
	background-color: rgb(255, 234, 217);
	border-radius: 18px;
}
</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <property name="leftMargin">
                     <number>15</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>10</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="SearchInput">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">#SearchInput{
	color: rgb(0, 0, 0);
	background-color: transparent;
}</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="frame">
                       <bool>false</bool>
                      </property>
                      <property name="placeholderText">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="SearchBtn">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>28</width>
                        <height>28</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>28</width>
                        <height>28</height>
                       </size>
                      </property>
                      <property name="baseSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">border-image: url(:/images/search.svg);</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTreeView" name="TreeView">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#TreeView{
    border: none;
    background-color: transparent;
	padding-right: 3px;
}

QTreeView::item:hover{
	background-color: rgb(255, 204, 175);
}

*{
	outline: none;
}

QTreeView::item:selected{
	color: black;
	background-color: rgb(255, 204, 175);
}

QHeaderView::section{
    background-color: rgb(255, 245, 239);
    height: 20px;
    border: none;
}

QScrollBar{
    background-color: rgb(255, 245, 239);
}

QScrollBar:vertical {
    width: 18px;
    margin-top: 20px;
}

QScrollBar:horizontal {
    background: transparent;
    height: 0px;
}

QScrollBar::handle {
    background-color: rgb(255, 172, 112);
    border-radius: 9px;
}

QScrollBar::add-line {
    width: 0px;
    height: 0px;
}

QScrollBar::sub-line{
    width: 0px;
    height: 0px;
}

QScrollBar::add-page {
	background: none;
}

QScrollBar::sub-page{
	background: none;
}

QLineEdit{

	background-color: rgb(255, 255, 255);
}</string>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="dragEnabled">
                  <bool>false</bool>
                 </property>
                 <property name="dragDropOverwriteMode">
                  <bool>false</bool>
                 </property>
                 <property name="dragDropMode">
                  <enum>QAbstractItemView::NoDragDrop</enum>
                 </property>
                 <property name="defaultDropAction">
                  <enum>Qt::IgnoreAction</enum>
                 </property>
                 <property name="animated">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="CollapseBtn">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>120</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#CollapseBtn{
	background-color: rgba(255, 155, 82, 50%);
	color: rgb(255, 255, 255);
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
	font: 16pt &quot;Microsoft YaHei UI&quot;;
}

#CollapseBtn:hover{
	background-color: rgba(255, 155, 82, 100%);
	border: 2px solid white;
	border-right: none;
}</string>
            </property>
            <property name="text">
             <string>◀</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="CollapsedPage">
         <property name="styleSheet">
          <string notr="true">#CollapsedPage{
	background-color: transparent;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ExpandBtn">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>120</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#ExpandBtn{
	background-color: rgba(255, 155, 82, 50%);
	color: rgb(255, 255, 255);
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
	font: 16pt &quot;Microsoft YaHei UI&quot;;
}
#ExpandBtn:hover{
	background-color: rgba(255, 155, 82, 100%);
	border: 2px solid white;
	border-left: none;
}</string>
            </property>
            <property name="text">
             <string>▶</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 216, 180);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
